File: app/src/main/java/room/model/ProviderWithCountry.java
package room.model;

import androidx.room.ColumnInfo;

public class ProviderWithCountry
{
    @ColumnInfo(name = "provider_id")
    private long providerId;

    @ColumnInfo(name = "provider_code")
    private String providerCode;

    @ColumnInfo(name = "provider_name")
    private String providerName;

    @ColumnInfo(name = "country_id")
    private long countryId;

    @ColumnInfo(name = "country_iso")
    private String countryIso;

    @ColumnInfo(name = "country_name")
    private String countryName;

    public ProviderWithCountry(long providerId, String providerCode, String providerName, long countryId, String countryIso, String countryName)
    {
        this.providerId = providerId;
        this.providerCode = providerCode;
        this.providerName = providerName;
        this.countryId = countryId;
        this.countryIso = countryIso;
        this.countryName = countryName;
    }

    public long getProviderId()
    {
        return this.providerId;
    }

    public String getProviderCode()
    {
        return this.providerCode;
    }

    public String getProviderName()
    {
        return this.providerName;
    }

    public long getCountryId()
    {
        return this.countryId;
    }

    public String getCountryIso()
    {
        return this.countryIso;
    }

    public String getCountryName()
    {
        return this.countryName;
    }
}

File: app/src/main/java/localhost/android/me/database/model/ProviderWithCountry.kt
package localhost.android.me.database.model

import androidx.room.ColumnInfo

class ProviderWithCountry(
    @ColumnInfo(name = "provider_id") private var providerId: Long,
    @ColumnInfo(name = "provider_code") private var providerCode: String,
    @ColumnInfo(name = "provider_name") private var providerName: String,
    @ColumnInfo(name = "country_id") private var countryId: Long,
    @ColumnInfo(name = "country_iso") private var countryIso: String,
    @ColumnInfo(name = "country_name") private var countryName: String
)
{
    fun getProviderId(): Long
    {
        return this.providerId
    }

    fun getProviderCode(): String
    {
        return this.providerCode
    }

    fun getProviderName(): String
    {
        return this.providerName
    }

    fun getCountryId(): Long
    {
        return this.countryId
    }

    fun getCountryIso(): String
    {
        return this.countryIso
    }

    fun getCountryName(): String
    {
        return this.countryName
    }
}
